storage-bigtable/src/bigtable.rs

```diff
@@ -897,6 +897,7 @@ mod tests {
                 rewards: Some(vec![]),
                 loaded_addresses: LoadedAddresses::default(),
                 return_data: Some(TransactionReturnData::default()),
+                compute_units_consumed: None,
```

Comment:
This should be set to a populated value in order to fully test block deserialization